vercel.json

```diff
@@ -25,7 +25,7 @@
     },
     {
       "source": "/breach",
-      "destination": "/reset",
+      "destination": "/using/id/guide-to-resets",
```

Comment:
```suggestion
      "destination": "/reset",
```

Reply:
this was used for the page about the network breach at the end of 2020, not for info about breaching, so should probably be left as-is (although that page doesn't work for some reason, dunno why though. I think it's missing a corresponding .js file)

Reply:
Gotcha, I've undone that change.